src/styles/index.css

```
@@ -0,0 +1,158 @@
+/* Variables, Global Styles  */
+:root {
+  --cream: #ffffff;
+  --bg_black: #121212;
+  --pure_black: #000000;
+  --navy_blue: #141721;
+  --hover_opaque: rgba(255, 255, 255, 0.3);
+  --modal_blackout: rgba(0, 0, 0, 0.7);
+}
+::-webkit-scrollbar {
+  background-color: #202324;
+  color: #aba499;
+  width: 0.6rem;
+}
+::-webkit-scrollbar-thumb {
+  background-color: #454a4d;
+}
+::-webkit-scrollbar-thumb:hover {
+  background-color: #575e62;
+}
+
+* {
+  font-family: 'Raleway', sans-serif;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-weight: normal;
+  font-family: inherit;
+  line-height: inherit;
+  background-color: var(--pure_black);
+  color: var(--cream);
+  -webkit-font-smoothing: antialiased;
+}
+
+html {
+  display: block;
+}
```

Whenever the home page is refreshed or loaded for the first time, the horizontal scrollbar is displayed for a second and hidden after the completion of the banner animation. You should hide the x-axis scrollbar to prevent it from showing up.
```suggestion
html {
  display: block;
   overflow-x: hidden;
}
```

---

This is one of those things I must've just seen so many times it went right over my head. Thanks!